config/locales/en.yml

```diff
@@ -60,7 +60,7 @@ en:
         name: "Biology"
         group: "Secondary"
         sub_head: "Biology - Secondary"
-        funding: "Bursaries of %{bursaries_postgraduate_biology} are available for trainee biology teachers if you're eligible."
+        funding: "Bursaries of %{bursaries_postgraduate_biology} are available for <a href='/life-as-a-teacher/explore-subjects/biology'>trainee biology teachers</a> if you're eligible."
```

Comment:
better to use double quotes for the attributes  - I think you can escape them like this:

```
funding: "Bursaries of %{bursaries_postgraduate_biology} are available for <a href=\"/life-as-a-teacher/explore-subjects/biology\">trainee biology teachers</a> if you're eligible."
```